.github/workflows/ci.yml

```diff
@@ -17,6 +17,15 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
+      # Cache node_modules
+      - name: Cache node_modules
+        uses: actions/cache@v3
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-node-${{ hashFiles('yarn.lock.json') }}
```

Comment:
Did you mean `yarn.lock` as opposed to `yarn.lock.json`?

Reply:
fixed, thanks for noticing this.